### DISCUSSION TITLE
Banner images not loading

### DISCUSSION THREAD

**Discussion by erayrafet** [rock5-images-repo/rock5-images-repo.github.io#1] [General]
![image](https://github.com/rock5-images-repo/rock5-images-repo.github.io/assets/34194191/f042565d-334c-4483-96e4-0d2f66aef938)

The terminal shows the links with a colon (:) at the end.

**Comment by erayrafet** [comment]
https://bugs.archlinux.org/task/62672

Appears to be an Arch Linux bug.

![image](https://github.com/rock5-images-repo/rock5-images-repo.github.io/assets/34194191/6b5c9e4b-a29e-47c5-bdda-c913d687bb92)

Solved by installing `qt5-imageformats`.

**Reply by kwankiu** [reply]
So this looks like some systems does not support the image format .webp by default, solution is to install `qt5-imageformats`.

**Reply by kwankiu** [reply]
Btw, Please report this to rpi-imager Arch Linux maintainer to add the following changes to the PKGBUILD:
```
# Support webp image format
- depends=('glibc' 'curl' 'hicolor-icon-theme' 'libarchive' 'openssl' 'qt5-base' 'qt5-declarative' 'qt5-quickcontrols2' 'qt5-svg' 'zlib')
+ depends=('glibc' 'curl' 'hicolor-icon-theme' 'libarchive' 'openssl' 'qt5-base' 'qt5-declarative' 'qt5-quickcontrols2' 'qt5-svg' 'qt5-imageformats' 'zlib')
```

**Reply by erayrafet** [reply]
They have a stupid account creation system for GitLab.

**Reply by kwankiu** [reply]
Yeah … I hope that someone already have an account there would help out …

**Comment by kwankiu** [comment]
Added ```qt5-imageformats``` dependency for [Radxa Imager](https://github.com/rock5-images-repo/radxa-imager).
